test/support/api/patients.js

```diff
@@ -31,3 +31,33 @@ Cypress.Commands.add('routePatient', (mutator = _.identity) => {
   })
     .as('routePatient');
 });
+
+Cypress.Commands.add('routePatients', (mutator = _.identity) => {
+  cy
+    .fixture('collections/patients').as('fxPatients')
+    .fixture('collections/groups').as('fxGroups');
+
+  cy.route({
+    url: '/api/patients**', // probably a regex to match for includes or just copy/past it?
+    response() {
+      const data = getResource(_.sample(this.fxPatients, 30), 'patients');
+      const groups = _.sample(this.fxGroups, 2);
```

Comment:
I think to make this a little more random we should sample 5 groups or so for the include and in the `_.each` do like `_.sample(groups, _.random(1, 3))`